src/main/java/com/feelfreecincinncati/dto/AdventureDTO.java

```diff
@@ -1,15 +1,15 @@
 package com.feelfreecincinncati.dto;
 
 public class AdventureDTO {
-	private int advID;
+	private int adventureID;

```

Comment:
Good idea.  I always spell out variable names entirely.  The time you save by abbreviating a name will quickly be lost when you try to remember which abbreviation you used.   :)